native.js

```diff
@@ -16,6 +16,7 @@ const bindingsByPlatformAndArch = {
 	linux: {
 		arm: { base: 'linux-arm-gnueabihf', musl: null },
 		arm64: { base: 'linux-arm64-gnu', musl: 'linux-arm64-musl' },
+		ppc64le: { base: 'linux-ppc64le-gnu', musl: null },
```

Comment:
@pavolloffay @lukastaegert 
I guess this `ppc64le` and `cpu` field in package.json need to be `ppc64`. `process.arch` seems to return `ppc64` instead of `ppc64le`.
https://github.com/konveyor/tackle2-ui/actions/runs/8150208242/job/22276088934#step:7:89

Reply:
Looks like you are right and `process.arch` always returns `ppc64` - which in practice means ppc64le as Node hasn't been supporting big-endian Linux for a while.  In theory there is also an `os.machine` interface that returns either `ppc64` or `ppc64le`, but it doesn't seem necessary to use that.

Reply:
If anyone wants to make an educated PR to improve here, this would be very welcome! Otherwise, I can also have a look in the next days.

Reply:
PR here: #5460

Reply:
Hi @sapphi-red, just wondering whether the native ppc64le rollup work fors you now?  It looks like tackle2-ui downgraded to rollup v3 in the meantime?

Reply:
I don't have a ppc64le machine so I don't know 😅

Reply:
I see, thanks!

Reply:
I spend some hours setting up emulated smoke tests for some environments including ppc64le and I can confirm it works now 😀
https://github.com/rollup/rollup/pull/5477